Livecheckables/libu2f-server.rb

```diff
@@ -1,4 +1,4 @@
 class Libu2fServer
-  livecheck :url   => "https://developers.yubico.com/libu2f-server/Releases",
+  livecheck :url   => "https://developers.yubico.com/libu2f-server/Releases/",
             :regex => /href="libu2f-server-([0-9\.]+)\.t/
```

Comment:
```suggestion
            :regex => /href="libu2f-server-(\d+(?:\.\d+)+)\.t/
```

As with previous comments, while we're updating this livecheckable, could you replace the (old) loose version-matching regex with the more explicit one we use now (as above)?

Reply:
Sounds good.